src/site/components/banner_alerts.drupal.liquid

```diff
@@ -0,0 +1,70 @@
+{% for entity in alert.entities %}
+
+{% assign alertType = entity.fieldAlertType %}
+
+{% if alertType = "information" %}
+{% assign alertType = "info" %}
```

Comment:
CS: SHouldn't these be indented inside the control structure?

Reply:
I'm not sure what the standard is, @swirtSJW - seems like we are all over the place in our liquid files. Fwiw, my VsCode liquid formatter thinks that this is the way it's supposed to look (e.g., indenting code within conditionals manually is pulled back flush after autofomatting).

Reply:
I think it makes sense to keep it indented like it was "normal" code. Ideally, we'd lint this, but we haven't gotten there yet. Without the indentation, it makes the following pretty tough to read.

Reply:
Updated @swirtSJW @cvalarida :+1: